scarlett/basics/talk.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: UTF-8 -*-
+
+# NOTE: Borrowed a great deal of knowledge from LISA project.
+
+import gobject
+gobject.threads_init()
+
+# Imports
+import scarlett
+import threading
+import os
+import gettext
+from Queue import Queue
+from time import sleep
+from subprocess import call
+# import urllib
+# from urllib import urlencode, urlopen
+from random import randint
+# from twisted.python import log
+from scarlett.basics import *
+from scarlett.constants import *
+import scarlett.basics.voice
+
+# System utterance definition : key : {(weight1, message1), (weight2, message2)}
+# Talk will randomly choose a message in list, balanced by weights
+_utterances = {
```

Comment:
I've been hearing a lot of good things about GStreamer 1.0; any perks in particular you've looked at?